packages/webview_flutter/example/lib/main.dart

```diff
@@ -125,6 +135,42 @@ class SampleMenu extends StatelessWidget {
     controller.evaluateJavascript(
         'Toaster.postMessage("User Agent: " + navigator.userAgent);');
   }
+
+  void _onClearCookies(
+      WebViewController controller, BuildContext context) async {
+    final String cookies =
+        await controller.evaluateJavascript('document.cookie');
```

Comment:
What do you think about adding a menu entry that just shows a toast with the cookies?
I'd feel more confident to verify that it's working if the button that's showing me the cookies is independent of the button that's deleting the cookies.

Reply:
Makes sense.